ClavaLaraExtraApi/.classpath

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER">
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" path="src-java"/>
-	<classpathentry kind="src" path="test"/>
-	<classpathentry kind="src" path="src-lara"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>
-	<classpathentry combineaccessrules="false" kind="src" path="/SymjaPlus"/>
-	<classpathentry combineaccessrules="false" kind="src" path="/ClavaAst"/>
-	<classpathentry combineaccessrules="false" kind="src" path="/jOptions"/>
-	<classpathentry combineaccessrules="false" kind="src" path="/SpecsUtils"/>
-	<classpathentry combineaccessrules="false" kind="src" path="/ClangAstParser"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
```

Comment:
Keep the project for now, do not need to add the JS file to the Clava API, this is WIP. Later we can move this to a devel branch for this feature

Reply:
Will be moved to a new branch and not reapplied to the main project. This is to prevent dead-code existing or being merged into staging or master.